package.json

```diff
@@ -6,33 +6,43 @@
   "bin": {
     "changelog-maker": "./changelog-maker.js"
   },
-  "author": "Rod <rod@vagg.org> (http://r.va.gg/)",
+  "author": "Rod <rod@vagg.org> (https://r.va.gg/)",
   "license": "MIT",
-  "dependencies": {
-    "async": "~2.6.1",
-    "chalk": "~2.4.1",
-    "commit-stream": "~1.1.0",
-    "debug": "~4.1.0",
-    "ghauth": "~3.2.1",
-    "ghissues": "~1.1.3",
-    "gitexec": "~1.0.0",
-    "list-stream": "~1.0.1",
-    "minimist": "~1.2.0",
-    "pkg-to-id": "~0.0.3",
-    "split2": "~3.0.0",
-    "strip-ansi": "~5.0.0"
-  },
   "repository": {
     "type": "git",
-    "url": "https://github.com/nodejs/changelog-maker.git"
+    "url": "git+https://github.com/nodejs/changelog-maker.git"
   },
+  "bugs": {
+    "url": "https://github.com/nodejs/changelog-maker/issues"
+  },
+  "homepage": "https://github.com/nodejs/changelog-maker#readme",
+  "keywords": [],
   "preferGlobal": true,
+  "dependencies": {
+    "async": "^3.1.0",
+    "chalk": "^2.4.2",
+    "commit-stream": "^1.1.0",
+    "debug": "^4.1.1",
+    "ghauth": "^3.2.1",
+    "ghissues": "^1.1.3",
+    "gitexec": "^2.0.0",
+    "list-stream": "^1.0.1",
+    "minimist": "^1.2.0",
+    "pkg-to-id": "0.0.3",
+    "split2": "^3.1.1",
+    "strip-ansi": "^5.2.0"
+  },
+  "devDependencies": {
+    "standard": "^14.3.1",
+    "tape": "^4.11.0"
+  },
   "scripts": {
     "lint": "standard",
-    "test": "standard && tape test.js"
+    "unit": "tape test.js",
+    "test": "npm run lint && npm run unit"
   },
-  "devDependencies": {
-    "standard": "~12.0.1",
-    "tape": "~4.9.1"
-  }
+  "files": [
```

Comment:
This is the only thing I'm not good with in this PR; you're excluding readme, license and tests in this, to only save 1071 bytes in the packed tarball. We should be shipping an entire, stand-alone package.

Reply:
First of all this change you are talking about is on #61 :P

Secondly, no, I don't exclude those. npm includes some stuff by default. I think you are confusing something there.

```
npm notice === Tarball Contents ===
npm notice 4.1kB changelog-maker.js
npm notice 1.9kB collect-commit-labels.js
npm notice 2.4kB commit-to-output.js
npm notice 488B  group-commits.js
npm notice 1.7kB groups.js
npm notice 371B  reverts.js
npm notice 1.2kB package.json
npm notice 1.1kB LICENSE.md
npm notice 4.7kB README.md
```

Reply:
so you're just excluding test.js, -1 from me, don't exclude anything, this project doesn't have anything of onerous size and is not intended to be a dependency for anything else